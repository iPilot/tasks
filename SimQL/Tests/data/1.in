{
    "data": {
        "a": {
            "x": 3.14,
            "b": [
                { "c": 15 },
                { "c": 9 }
            ]
        },
        "z": [ 2.65, 35 ]
    },
    "queries": [ "sum(a.b.c)", "min(z)", "max(a.x)", "z.s", "max(a.z)", "z", "a.b.c" ],
}