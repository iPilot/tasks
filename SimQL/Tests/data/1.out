sum(a.b.c) = 24
min(z) = 2.65
max(a.x) = 3.14
z.s
max(a.z)
z = 2.65
a.b.c = 15